<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowMenu</class>
 <widget class="QMainWindow" name="MainWindowMenu">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ForRorfloLisp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1230</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="file_last_projects_menu">
     <property name="title">
      <string>Открыть последние</string>
     </property>
    </widget>
    <widget class="QMenu" name="menu_9">
     <property name="title">
      <string>Создать</string>
     </property>
     <addaction name="file_create_project_action"/>
    </widget>
    <widget class="QMenu" name="menu_10">
     <property name="title">
      <string>Открыть</string>
     </property>
     <addaction name="file_open_project_action"/>
    </widget>
    <addaction name="menu_9"/>
    <addaction name="menu_10"/>
    <addaction name="file_last_projects_menu"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="file_save"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="file_save_project_action"/>
    <addaction name="separator"/>
    <addaction name="action_11"/>
    <addaction name="file_settings_action"/>
    <addaction name="separator"/>
    <addaction name="file_close_project_action"/>
    <addaction name="file_exit_action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="edit_undo"/>
    <addaction name="edit_redo"/>
    <addaction name="separator"/>
    <addaction name="edit_cut"/>
    <addaction name="edit_copy"/>
    <addaction name="edit_put"/>
    <addaction name="edit_delete"/>
    <addaction name="separator"/>
    <addaction name="edit_ctrl_all"/>
    <addaction name="separator"/>
    <addaction name="action_23"/>
    <addaction name="action_24"/>
    <addaction name="separator"/>
    <addaction name="action_26"/>
    <addaction name="action_27"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="separator"/>
    <addaction name="view_project_files_action"/>
    <addaction name="view_interpretator_action"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="project_menu">
    <property name="title">
     <string>Проект</string>
    </property>
    <addaction name="project_settings_action"/>
   </widget>
   <widget class="QMenu" name="interpretator_menu">
    <property name="title">
     <string>Интерпретатор</string>
    </property>
    <widget class="QMenu" name="interpretator_sexpr_menu">
     <property name="title">
      <string>Отправить S-выражение</string>
     </property>
    </widget>
    <addaction name="interpretator_start_action"/>
    <addaction name="interpretator_stop_action"/>
    <addaction name="interpretator_sexpr_menu"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="interpretator_add_action"/>
    <addaction name="interpretator_edit_action"/>
    <addaction name="separator"/>
    <addaction name="interpretator_clear_action"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="project_menu"/>
   <addaction name="interpretator_menu"/>
   <addaction name="menu_6"/>
  </widget>
  <action name="action_2">
   <property name="text">
    <string>Новое окно</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>какие-то</string>
   </property>
  </action>
  <action name="file_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Сохранить все</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Автосохранение</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>тоже чё-то</string>
   </property>
  </action>
  <action name="file_exit_action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="edit_undo">
   <property name="text">
    <string>Отменить</string>
   </property>
  </action>
  <action name="edit_redo">
   <property name="text">
    <string>Повторить</string>
   </property>
  </action>
  <action name="edit_cut">
   <property name="text">
    <string>Вырезать </string>
   </property>
  </action>
  <action name="edit_copy">
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="edit_put">
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>Найти</string>
   </property>
  </action>
  <action name="action_24">
   <property name="text">
    <string>Заменить</string>
   </property>
  </action>
  <action name="action_26">
   <property name="text">
    <string>Найти в файлах</string>
   </property>
  </action>
  <action name="action_27">
   <property name="text">
    <string>Заменить в файлах</string>
   </property>
  </action>
  <action name="view_project_files_action">
   <property name="text">
    <string>Файлы проекта</string>
   </property>
  </action>
  <action name="view_interpretator_action">
   <property name="text">
    <string>Консоль</string>
   </property>
  </action>
  <action name="interpretator_start_action">
   <property name="text">
    <string>Запустить</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Настройка пользовательских команд</string>
   </property>
  </action>
  <action name="project_settings_action">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="file_create_project_action">
   <property name="text">
    <string>Проект</string>
   </property>
  </action>
  <action name="interpretator_stop_action">
   <property name="text">
    <string>Остановить</string>
   </property>
  </action>
  <action name="file_open_project_action">
   <property name="text">
    <string>Проект</string>
   </property>
  </action>
  <action name="interpretator_add_action">
   <property name="text">
    <string>Добавить интерпретатор</string>
   </property>
  </action>
  <action name="interpretator_edit_action">
   <property name="text">
    <string>Редактировать интерпретатор</string>
   </property>
  </action>
  <action name="file_close_project_action">
   <property name="text">
    <string>Закрыть проект</string>
   </property>
  </action>
  <action name="file_save_project_action">
   <property name="text">
    <string>Сохранить проект</string>
   </property>
  </action>
  <action name="interpretator_clear_action">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="actionq">
   <property name="text">
    <string>q</string>
   </property>
  </action>
  <action name="file_settings_action">
   <property name="text">
    <string>Настройки...</string>
   </property>
  </action>
  <action name="edit_delete">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="edit_ctrl_all">
   <property name="text">
    <string>Выделить всё</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
